src/zampy/cli.py

```diff
@@ -0,0 +1,20 @@
+"""Implements CLI interface for Zampy."""
+import click
+from zampy.recipe import RecipeManager
+
+
+@click.command()
+@click.option(
+    "--filename",
+    prompt="Path to the recipe filename",
+    help="Path to the recipe filename.",
+)
```

Comment:
can we have something like this instead:
```
@click.argument("recipe", type=click.Path(exists=True, path_type=Path))
```

Reply:
Ok! Thanks for the suggestion.